middle_end/flambda/types/env/aliases.ml

```diff
@@ -16,25 +16,40 @@
 
 module Const = Reg_width_things.Const
 
+type coercion_to_canonical = Coercion.t
+type map_to_canonical = coercion_to_canonical Name.Map.t
```

Comment:
What about introducing a new module Map_to_canonical here? [map_inter] and [map_union] go directly under it